jetpack.php

```diff
@@ -15,7 +14,6 @@
  * Domain Path: /languages/
  */
 
-require plugin_dir_path( __FILE__ ) . '/vendor/autoload.php';
```

Comment:
Seems like our autoloader doesn't handle legacy classmapping properly? When this line is removed, I get a:

```
Fatal error: Uncaught Error: Class 'Jetpack_Options' not found in .../jetpack/packages/connection/src/Manager.php on line 410
```

Reply:
da968676800c53a842c29248cd270e6e159e7564 resolved this 👍